server/src/resolvers/trainingResolver.ts

```diff
@@ -69,14 +71,66 @@ const TrainingResolvers = {
     submitModule: async (
       _parent: any,
       args: { moduleID: number; answerSheet: AnswerInput[] },
-      { ifAllowed }: ApolloContext
+      { ifAllowed, user }: ApolloContext
     ) => {
       return ifAllowed(
         [Privilege.ADMIN, Privilege.LABBIE, Privilege.MAKER],
         async (user) => {
-          // TODO: grade answer sheet
-          console.log(args.answerSheet);
-          return null;
+          const module: any = await (
+            await ModuleRepo.getModuleByID(args.moduleID)
+          ).quiz;
```

Comment:
Would this work?

``` ts
const  { quiz } = await ModuleRepo.getModuleByID(args.moduleID);
```

Reply:
yes but I needed to add a type for the `quiz` column in the TrainingModuleRow interface

Reply:
In my example, `quiz` should come back as an `object` type because of https://github.com/rit-construct-makerspace/makerspace/blob/master/server/src/db/tables.ts#L93, no?

The way you have it is fine tbh. Just curious.